global.json

```diff
@@ -1,11 +1,11 @@
 {
   "sdk": {
-    "version": "6.0.100",
+    "version": "6.0.200",
     "allowPrerelease": true,
     "rollForward": "latestPatch"
   },
   "tools": {
-    "dotnet": "6.0.100",
+    "dotnet": "6.0.200",
```

Comment:
❔ Why update this instead of just updating to Microsoft.Net.Compilers.Toolset package?

Reply:
You might be teaching me something. If we set the Toolset package version will the SDK pick that up and use those compilers for the build?

Reply:
> If we set the Toolset package version will the SDK pick that up and use those compilers for the build?

Yes.

Reply:
> ❔ Why update this instead of just updating to Microsoft.Net.Compilers.Toolset package?

This would also be a reasonable way of doing it. It seems also beneficial to move forward to stable SDKs as they come out (finding out earlier if there are issues adopting it).